inst/models/pk_pegasparaginase_modifiedwurthwein.json5

```
@@ -0,0 +1,109 @@
+{
+  "id": "pk_pegasparaginase_wurthwein",
```

We previously discussed using the `modified` tag since this model has been modified from the original. Did you want to include that in this version of the model?

---

Great catch! Altered.

---

we should rename this to be consistent with the file name
```suggestion
  "id": "pk_pegasparaginase_modifiedwurthwein",
```